src/main/java/org/opensearch/knn/index/query/KNNQueryFactory.java

```diff
@@ -102,38 +105,30 @@ public static Query create(CreateQueryRequest createQueryRequest) {
         }
     }
 
-    private static Query getKnnByteVectorQuery(String indexName, String fieldName, byte[] byteVector, int k, Query filterQuery) {
-        if (filterQuery != null) {
-            log.debug(
```

Comment:
not sure if that is critical, but did you check that logs we're removing here are not used?

Reply:
We don't need a log if it is query with filter or not as underline method is same.
Let me add this line back though. `log.debug(String.format("Creating Lucene k-NN query for index: %s \"\", field: %s \"\", k: %d", indexName, fieldName, k));`